scvi/__init__.py

```diff
@@ -3,12 +3,11 @@
 # Set default logging handler to avoid logging with logging.lastResort logger.
 import logging
 
+# this import needs to come after prior imports to prevent circular import
+from . import data, external, model, utils
```

Comment:
this needs to be kept after. Perhaps a no QA comment if it's an issue?

Reply:
Yeah flake8 autofixed it for some reason...not sure why but when changing it back without noqa it seems to pass

Reply:
I would still add a noqa so it doesn't do it again